styles/global.css

```diff
@@ -37,6 +37,10 @@ html {
   transition: background-color 0.1 ease-in-out;
 }
 
+.skip-nav {
+  @apply absolute px-4 py-2 transition-transform duration-200 transform -translate-y-12 focus:translate-y-3 -top-8;
```

Comment:
Doesn't it still need the sr classes?

Reply:
The sr classes are mainly used just to hide and show an element visually without fully hiding them from a screen reader. [https://tailwindcss.com/docs/screen-readers](https://tailwindcss.com/docs/screen-readers).